spec/controllers/coronavirus_landing_page_controller_spec.rb

```diff
@@ -25,5 +25,54 @@
         expect(response.headers["Cache-Control"]).to eq("max-age=#{30.seconds}, public")
       end
     end
+
+    context "when testing national_applicability" do
+      render_views
+
+      before do
+        allow(Rails.env).to receive(:production?).and_return(true)
```

Comment:
This should only belong in the first test as the other tests should all fail if Rails has a production environment. I'm a bit confused why they're not so not sure if I've missed something.

Reply:
We do need it this, otherwise, it will load the content item from `CoronavirusTimelineNationsContentItem` rather than the live one we're stubbing.

For most of the tests it doesn't matter as the content item loaded for `CoronavirusTimelineNationsContentItem` is the same as the fixture for `coronavirus_content_item_with_timeline_national_applicability`, but that's not true of `coronavirus_content_item_with_timeline_national_applicability_without_wales`, so that test fails.